src/window.rs

```diff
@@ -473,17 +476,18 @@ where
                 }
                 WindowEvent::ScaleFactorChanged {
                     scale_factor,
-                    new_inner_size,
+                    inner_size_writer: _,
                 } => {
                     // Ensure both values are updated before any behavior
                     // callbacks are invoked.
                     self.scale = scale_factor;
-                    let inner_size_changed = self.inner_size != new_inner_size;
-                    self.inner_size = new_inner_size;
-                    behavior.scale_factor_changed(self);
-                    if inner_size_changed {
-                        behavior.resized(self);
-                    }
+                    // TODO not sure how to implement now
+                    // let inner_size_changed = self.inner_size != new_inner_size;
+                    // self.inner_size = new_inner_size;
+                    // behavior.scale_factor_changed(self);
+                    // if inner_size_changed {
+                    //     behavior.resized(self);
+                    // }
```

Comment:
Not really sure what should happen here now

Reply:
I don't think we should change this. We previously didn't support using the mutable state, there's no reason to adopt it now.